config/default-example.yaml

```diff
@@ -19,22 +19,6 @@ logger:
   archive: true
 
 dataSources:
-  dataSources: ['json', 'oracledb', 'awsS3']
-  json:
-    dbPath: 'tests/unit/mock-data.json'
-  oracledb:
-    connectString: ${DB_URL}
-    user: ${DB_USER}
-    password: ${DB_PASSWD}
-    poolMin: 4
-    poolMax: 4
-    poolIncrement: 0
-  awsS3:
-    bucket: bucketName
-    apiVersion: '2006-03-01'
-    accessKeyId: ${ACCESS_KEY_ID}
-    secretAccessKey: ${SECRET_ACCESS_KEY}
-    region: us-west-2
-    # These values are for local or proxy S3 instances
-    endpoint: null
-    s3ForcePathStyle: false
+  dataSources: ['ldap']
+  ldap:
+    url: 'ldap://directory.oregonstate.edu:389'
```

Comment:
We should use `ldaps`portocol. In addition, we need to make sure we are connecting to `ldap.onid.orst.edu` because that is the name in the SSL certificate. By using these, I don't think port will be necessary.